excel.c

```diff
@@ -2380,12 +2401,12 @@ zend_bool php_excel_write_cell(SheetHandle sheet, BookHandle book, int row, int
 
 		case IS_STRING:
 			data_zs = Z_STR_P(data);
-			if (dtype == PHP_EXCEL_STRING) {
-				return xlSheetWriteStr(sheet, row, col, (const char*) ZSTR_VAL(data_zs), format);
-			}
 			if (Z_STRLEN_P(data) > 0 && '\'' == Z_STRVAL_P(data)[0]) {
 				return xlSheetWriteStr(sheet, row, col, (const char*) ZSTR_VAL(data_zs) + 1, format);
 			}
+			if (dtype == PHP_EXCEL_STRING) {
+				return xlSheetWriteStr(sheet, row, col, (const char*) ZSTR_VAL(data_zs), format);
+			}
```

Comment:
I see you have changed the ordering of the check for dtype == PHP_EXCEL_STRING. This reordering causes a regression for me: with type PHP_EXCEL_STRING I don't expect any "magic", the value should be copied as-is, so values starting with a single quote should also have the single quote in their value, but with this change the single quote will be stripped.

Furthermore I would have expected this change in a seperate commit, as this has nothing to do with the new methods for libXL 3.8.0.